### ISSUE TITLE
GitHub file size badge fails when file is >1Mb

### ISSUE THREAD

**Issue by komandos84** [badges/shields#4811]
Are you experiencing an issue with...

- [x] [shields.io](https://shields.io/#/)

:beetle: **Description**

**for that order**
<a  href="https://ifotos.pl/z/qaspapn/"><img src="https://s6.ifotos.pl/img/1png_qaspapn.png"></a>

**zip file are not recognizing**

<a  href="https://ifotos.pl/z/qaspape/"><img src="https://s6.ifotos.pl/img/2png_qaspape.png"></a>

for other type of file like txt or md everythink is fine.

:link: **Link to the badge**

![GitHub file size in bytes](https://img.shields.io/github/size/komandos84/wot_sounds/JakWWII_crew_sounds/zip/JakWW2.zip?style=for-the-badge)

:bulb: **Possible Solution**

**Comment by chris48s** [comment]
The reason its not reporting the size isn't because its a zip file. We use the GH contents API for this badge and it won't return a result for files > 1Mb: https://api.github.com/repos/komandos84/wot_sounds/contents/JakWWII_crew_sounds/zip/JakWW2.zip

> Any solutions?

There's no workaround for that while we rely on the v3 contents endpoint, but I wonder if we could work around this by switching the badge to use the GH v4 API and looking at `blob.byteSize`? https://developer.github.com/v4/object/blob/#blob (unconfirmed)

**Comment by jacobtomlinson** [comment]
@chris48s I think this can be closed, it seems the issue was resolved.

Getting the size of [this large file in scipy](https://github.com/scipy/scipy/blob/main/benchmarks/benchmarks/qapdata/qap_probs.npz) seems to work fine. 

```
https://img.shields.io/github/size/scipy/scipy/benchmarks%2Fbenchmarks%2Fqapdata%2Fqap_probs.npz
```

![GitHub file size in bytes](https://img.shields.io/github/size/scipy/scipy/benchmarks%2Fbenchmarks%2Fqapdata%2Fqap_probs.npz)

**Comment by chris48s** [comment]
Yeah this restriction has been lifted on GitHub's side. I reckon this probably happened when they did this https://github.blog/changelog/2022-05-03-increased-file-size-limit-when-retrieving-file-contents-via-rest-api/
